ApeSwapFinance/frontend
# [FEAT] - Homepage LHD Swiper Slider

## ⛑️ **What was done:**
- Added LHD Swiper Slider to home page.

### 🧪 **How to test:**
- Visit homepage
- "Learn More" button should navigate to: https://apeswap.gitbook.io/apeswap-finance/product-and-features/liquidity-health-dashboard
- "Join Beta" Button should navigate to `/liquidity-dashboard`
- Counter should animate on first view
- A random LHD image should be rendered on the top right of the LHD chart.
 Web:
- Click on the carousel button
Mobile:
- Swipe to the right 

### **Demo :**
![Screenshot 2023-07-08 at 12 52 10](https://github.com/ApeSwapFinance/frontend/assets/47801291/952e84d7-5e2d-4aff-85d0-b9310d791800)
![Screenshot 2023-07-08 at 12 52 34](https://github.com/ApeSwapFinance/frontend/assets/47801291/78616145-2f10-4e3c-8719-520b85ed69cc)
